### DISCUSSION TITLE
Alerter for Stack events?

### DISCUSSION THREAD

**Discussion by parnic** [moghtech/komodo#162] [Q&A] [answered]
Is it possible to create an alerter for Stack state changed? I don't see Stacks listed in either the Alert Types or Resource lists for an alerter, but there are Stack alerts that show up on the Alerts page.

**Comment by mbecker20** [comment]
I think the UI wasn't updated to configure this on Alerters, thanks for the heads up.

**Comment by mbecker20** [comment, marked as answer]
I just pushed out frontend only fix to configure these: https://github.com/mbecker20/komodo/commit/97ced3b2cb43b31a47957c034c61b6bec228bbfb. Just update to the latest image and try it again.

**Reply by parnic** [reply]
Thank you!
